src/Navigation/styled.js

```diff
@@ -26,18 +26,11 @@ export const Icon = styled.img`
 
 export const Title = styled.h1`
   margin: 0;
-  font-weight: 400;
+  font-weight: 500;
   font-size: 24px;
   letter-spacing: -1.5px;
   display: flex;
   align-items: center;
-  flex-shrink: 0;
```

Comment:
I'm not sure why the deleted styles were here - maybe it appeared after merge? Anyways, I deleted them.